test/test_registrar_db.py

```diff
@@ -13,7 +13,7 @@
 # BEGIN TEST DATA
 
 test_data = {
-    'agent_id': 'D432FBB3-D2F1-4A97-9EF7-75BD81C00000',
+    'agent_id': '4d32fbb3-d2f1-4a97-9ef7-75bd81c00000',
```

Comment:
This isn't just a lower-case, it actually changes the first 2 characters from "D4" to "4d". @lukehinds This is the cause behind the failure in the `test/test_registrar_db.py` CI test

Reply:
Ah, well spotted, my bad, no idea how I managed that used the vscode lower case function.